extras/ams_wrapper/src/api/models/classification_attributes_model.py

```diff
@@ -58,24 +58,25 @@ def postprocess_inference_output(self, inference_output: dict) -> str:
                     highest_prob = probability
 
                 if is_softmax or is_softmax is None:
-                    attribute = Attribute(class_name, probability)
+                    tag = Tag(class_name, probability)
                 else:
                     value = probability * value_multiplier
-                    attribute = Attribute(class_name, value)
+                    tag = Tag(class_name, value)
 
-                attributes.append(attribute)
+                tags.append(tag)
 
-            attributes.sort(key=lambda attr: attr.confidence, reverse=True)
+            tags.sort(key=lambda attr: attr.confidence, reverse=True)
             if current_conf.top_k_results:
-                attributes = attributes[:current_conf.top_k_results]
+                tags = tags[:current_conf.top_k_results]
             if current_conf.confidence_threshold:
-                attributes = [attr for attr in attributes
-                              if attr.confidence >= current_conf.confidence_threshold]
-            classification = SingleClassification(subtype_name=output_name, attributes=attributes)
-            classifications.append(classification)
+                tags = [attr for attr in tags
+                        if attr.confidence >= current_conf.confidence_threshold]
+            if len(tags) != 0:
```

Comment:
To be consistent with detection model you could go with:
```
if not tags:
    return None
```
Then the main loop shall return 204 in case there's no classification above confidence threshold.

Reply:
Committed the changes. kindly review the updated file